### PR TITLE
add a code of conduct

### PR DESCRIPTION
**Pull request by goern**
Signed-off-by: Christoph Görn <goern@redhat.com>

### CODE REVIEW

**Review by fridex — COMMENTED** [review]
/lgtm

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Project maintainers have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are not
+aligned to this Code of Conduct. By adopting this Code of Conduct, project maintainers
+commit themselves to fairly and consistently applying these principles to every aspect
+of managing this project. Project maintainers who do not follow or enforce the Code of
+Conduct may be permanently removed from the project team.
+
+This code of conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community. All repositories
+unter https://github.com/thoth-station/ are considered the project or its community.
```
Comment (tlegen-k) [inline CODE_OF_CONDUCT.md:32]: *under

**Review by tlegen-k — APPROVED** [review]

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+sexual orientation, disability, personal appearance, body size, race, ethnicity, age,
+religion, or nationality.
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery
+* Personal attacks
+* Trolling or insulting/derogatory comments
+* Public or private harassment
+* Publishing others' private information, such as physical or electronic addresses,
```
Comment (xtuchyna) [inline CODE_OF_CONDUCT.md:19]:
```suggestion
* Publishing private information of others, such as physical or electronic addresses,
```

**Review by xtuchyna — COMMENTED** [review]
just one comment, otherwise lgtm

**Review by tlegen-k — APPROVED** [review]
/lgtm

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+commit themselves to fairly and consistently applying these principles to every aspect
+of managing this project. Project maintainers who do not follow or enforce the Code of
+Conduct may be permanently removed from the project team.
+
+This code of conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community. All repositories
+under https://github.com/thoth-station/ are considered the project or its community.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior in Project Thoth may
+be reported by contacting the <aicoe-thoth@redhat.com>.
```
Comment (4n4nd) [inline CODE_OF_CONDUCT.md:35]: `the` before the email address doesn't make sense, maybe change it to `at <aicoe-...>`?

**Pull request merged**